samcli/commands/init/interactive_init_flow.py

```diff
@@ -305,6 +323,25 @@ def _get_app_template_properties(
     return (runtime, base_image, package_type, dependency_manager, template_chosen)
 
 
+def prompt_user_to_enable_tracing():
+    """
```

Comment:
docstring seems not matching what the function actually accepts and returns. Can you please update it?